src/css/style.scss

```
@@ -11,6 +11,11 @@
   text-align: right;
 }
 
+.usa-table tfoot td {
+  font-weight: bold;
+  background-color: #dedede;
```

Could avoid hard-coding and use the token instead:

```suggestion
  background-color: color('base-lighter');
```

Via: https://design.login.gov/utilities/color/